testsuite/basic/src/test/java/io/smallrye/jwt/KeyLocationResolverTest.java

```diff
@@ -76,13 +76,23 @@ public void testVerifyWithJwkKeyWithNonMatchingKidFromSet() throws Exception {
     }
 
     @Test
-    public void testVerifyWithPemKey() throws Exception {
+    public void testVerifyWithClassPathPemKey() throws Exception {
         verifyToken("key3", null, "publicKey.pem");
     }
 
     @Test
-    public void testVerifyWithPemKeyWithMatchingKid() throws Exception {
-        verifyToken("key3", null, "publicKey.pem");
+    public void testVerifyWithClassPathPemKey2() throws Exception {
+        verifyToken("key3", null, "classpath:publicKey.pem");
+    }
+
+    @Test
+    public void testVerifyWithFileSystemPemKey() throws Exception {
+        verifyToken("key3", null, "target/test-classes/publicKey.pem");
```

Comment:
There's no prefix here, so it should be a classpath resource, but `target/test-classes/publicKey.pem` seems more like a filesystem path?

Reply:
Aha, that's intentional. All good :-)